documentation/dev/src/explore/design/ledger/pos-integration.md

```diff
@@ -53,48 +49,49 @@ For slashing tokens, we implement a [PoS slash pool account](vp.md#pos-slash-poo
 
 The validator transactions are assumed to be applied with an account address `validator_address`.
 
-- `become_validator(consensus_key, staking_reward_address)`:
+- `become_validator(consensus_key, commission_rate, max_commission_rate_change)`:
   - creates a record in `validator/{validator_address}/consensus_key` in epoch `n + pipeline_length`
-  - creates a record in `validator/{validator_address}/staking_reward_address`
+  - creates a record in `validator/{validator_address}/commission_rate` in epoch `n + pipeline_length`
```

Comment:
Does `commission_rate` need to be epoched? Currently when the inflation is applied, the commission rate is read from storage as a single value, then it is applied to store the delegator rewards product, which is epoched and contains the commission rate information inherently.

Reply:
I see, good point! Let's just use a single value and document why

Reply:
I thought more about this... I now think it is right to be epoched to ensure that this can only change by at most `max_commission_rate_change` per epoch.

Reply:
ah, right! I guess it doesn't need to be fully "epoched", but for this it may be convenient